### DISCUSSION TITLE
errorin arch linux AUR pacage : install: cannot stat 'debian/opensnitch.logrotate

### DISCUSSION THREAD

**Discussion by Maziar123** [evilsocket/opensnitch#752] [General]
HI
 
install in arch Linux opensnitch-git opensnitch-ebpf-module-git

opensnitch-git :
error : install: cannot stat 'debian/opensnitch.logrotate': No such file or directory

**Comment by gustavo-iniguez-goya** [comment]
That's probably due to the latest change 564c263 

Now those files are under: utils/packaging/daemon/deb/debian/ https://github.com/evilsocket/opensnitch/tree/master/utils/packaging/daemon/deb/debian

**Comment by Maziar123** [comment]
Any fix ?

**Reply by gustavo-iniguez-goya** [reply]
The AUR package maintainer(s) should update the package  opensnitch-git. Probably it'll be fixed soon, but someone has to report it :)

**Comment by Maziar123** [comment]
I report it last day
